scripts/initializers/index.js

```diff
@@ -49,6 +50,7 @@ export default async function initializeDropins() {
   events.enableLogger(true);
   // Set Fetch Endpoint (Global)
   setEndpoint(await getConfigValue('commerce-core-endpoint'));
+  setFetchGraphQlHeaders(await getHeaders('all'));
```

Comment:
FWIW in the future, the `getHeaders(scope)` function will probably [contain this functionality](https://github.com/hlxsites/aem-boilerplate-commerce/pull/286/files#diff-eb55b6bcdfbd2110ceed72aa446694d87208d331fbcf59d2887701c142611ccfR95).

As in, `all` scope is automatically applied.

That said, you will not have that code here in this repo, most likely, since that was a PR against boilerplate. So this is fine, or you may want to consider doing two things:

1) Modify `getHeaders` function to also spread `all` scope headers in response
2) Ensure using `getHeaders()` in some way, wherever necessary (such as here)